packages/plugin-ext-vscode/src/node/plugin-vscode-deployer-participant.ts

```diff
@@ -26,7 +26,7 @@ export class PluginVSCodeDeployerParticipant implements PluginDeployerParticipan
 
     async onWillStart(context: PluginDeployerStartContext): Promise<void> {
         const extensionsDirUri = await this.environments.getExtensionsDirUri();
-        context.userEntries.push(extensionsDirUri.withScheme('local-dir').toString());
+        context.userEntries.push(extensionsDirUri.withScheme('local-dir').toString(true));
```

Comment:
Are you sure it is necessary?

Reply:
You are right, it works without. Will remove.

Reply:
Is there a reason why we prefer URI encoded strings? It made goofy-looking paths when debugging, and if someone doesn't pass it back into a URI object it will most likely cause issues. I could not find a reason why we want URI encoded strings here.

Reply:
We use encoded by default since we pass them over the wire and not encoded when we need to give to a user for navigation. Here it does not matter. What is important that one should use `FileUri.parse` to get a system dependent path it can handle encoded and not encoded URIs.